themes/prism-funky.css

```diff
@@ -115,3 +115,16 @@ code[class*="language-"] {
 .token.deleted {
 	color: red;
 }
+
+/* Plugin styles: Diff Highlight */
+pre.diff-highlight > code .token.deleted:not(.prefix),
+pre > code.diff-highlight .token.deleted:not(.prefix) {
+	background-color: rgba(255, 0, 0, .3) !important;
+	display: inline !important;
+}
+
+pre.diff-highlight > code .token.inserted:not(.prefix),
+pre > code.diff-highlight .token.inserted:not(.prefix) {
+	background-color: rgba(0, 255, 128, .3) !important;
+	display: inline !important;
```

Comment:
Instead of `!important`, we can double the class names for extra specificity like [I did here](https://github.com/PrismJS/prism/blob/master/themes/prism-coy.css#L202-L212).  That gives consumers more flexibility if they need to modify the styles.

Reply:
Ooof. Very well then.

Reply:
Done.